stepactions/bundles/get-unreleased-bundle/0.1/README.MD

```diff
@@ -0,0 +1,26 @@
+# get-unreleased-bundle stepaction
+
+This StepAction retrieves the highest bundle version from a specified package and channel in the provided FBC fragment.
+If package name and/or channel name are not specified, the step will determine the default package and/or channel name.
+
+Default package name determination:
+* If there is only one 'olm.package', it's name is returned
+* If multiple 'olm.package' entries contain unreleased bundles, user input is required; the PACKAGE_NAME parameter must be set by the user
+
+Default Channel Name Determination:
+* The default channel name corresponds to the 'defaultChannel' entry of the selected package
+
+The StepAction checks whether the bundle is unreleased, specifically, not present in the Red Hat production Index Image (registry.redhat.io/redhat/redhat-operator-index).
```

Comment:
Nit: I would say `the highest bundle version`  like you did above: 

`The StepAction checks whether the highest bundle version is unreleased`